src/ray/protobuf/node_manager.proto

```diff
@@ -24,9 +24,11 @@ message WorkerLeaseReply {
 message ReturnWorkerRequest {
   // Port of the leased worker that we are now returning.
   int32 worker_port = 1;
+  // Pid of the leased worker we are now returning.
+  int32 worker_pid = 2;
```

Comment:
could we change to `worker_id` instead? since `pid` no longer identifies a worker with multi-threading.